### PR TITLE
Homework9: Alina Shautsova

### PR DESCRIPTION
**Pull request by AlinaShautsova**

### CODE REVIEW

File: homework9_count_numbers_of_letters.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    for i in range(1, len(line)):
+        if line[i] == line[i - 1]:
+            counter += 1
+        else:
+            if counter > 1:
+                answer.append(line[i - 1] + str(counter))
+            else:
+                answer.append(line[i - 1])
+            counter = 1
+    return count_last_letter(line, answer, counter)
```
Comment (Stml89) [inline homework9_count_numbers_of_letters.py]: separate function to work with the last character?

File: homework9_count_numbers_of_letters.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    else:
+        answer.append(line[-1])
+    return "".join(answer)
+
+
+assert count_the_number_of_letter("cccbba") == "c3b2a"
+assert count_the_number_of_letter("abeehhhhhccced") == "abe2h5c3ed"
+assert count_the_number_of_letter("aaabbceedd") == "a3b2ce2d2"
+assert count_the_number_of_letter("abcde") == "abcde"
+assert count_the_number_of_letter("aaabbdefffff") == "a3b2def5"
```
Comment (Stml89) [inline homework9_count_numbers_of_letters.py]: it is a good practice to write a message if `assert` happens

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: homework9_count_numbers_of_letters.py (hunk 1 above)
Comment (AlinaShautsova) [reply homework9_count_numbers_of_letters.py]: Yes, it was a creative approach to the solution :)
Changed it.

File: homework9_count_numbers_of_letters.py (hunk 2 above)
Comment (AlinaShautsova) [reply homework9_count_numbers_of_letters.py]: Done)

**Review by Stml89 — APPROVED** [review]
Please merge all approved PRs

**Pull request merged**